CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ## [Unreleased]
 
+### Breaking changes
+- `objects.layout` use `flexbox` instead of `inline-block` so we remove `o-layout__item` and `o-layout--stretch` because they are no longer needed. [[#380](https://github.com/inuitcss/inuitcss/issues/380)]
```

Comment:
`use` -> `uses`.

But possibly this whole changelog entry can be removed, if we decide to keep the `.o-layout__item` element class and the `--stretch` modifier.

Reply:
Since 2018, I update my personal grid system and I think using flex box utilizes is better than lot of modifier in any object. I will share my approach as soon as possible.